AEM/mlperf/README.md

```diff
@@ -7,9 +7,720 @@ translation, and others. The benchmark is representative of real-world
 workloads and as a fair and useful way to compare the performance of different
 machine learning systems.
 
+In this document, we'll show how to run Intel MLPerf v3.1 submission with Intel
+optimized Docker images.
```

Comment:
please provide the 3.1 github repo directly since it is an IDZ article, and roughly mention the important folders such as automation to them.
https://github.com/mlcommons/inference_results_v3.1